README.md

```diff
@@ -1,31 +1,38 @@
-#### Create a Jazz Lyric - Python Web App using Flask
+## Create a Jazz Lyric - Python Web App using Flask
 
 The web app lets you:
 - create a jazz lyric
 - see the jazz lyric & reports the number of vowels in the lyric
 - see a jazz song comprised of all the previously-created jazz lyrics
 
-View a deployed version at:  
-http://kimlew.pythonanywhere.com/
+View a deployed version at: `http://kimlew.pythonanywhere.com/`
 
 This app:
 - has & uses Flask function decorators
 - uses the Jinja2 text template engine
 
 This app has been tested with:
-- Python 3.9	
-- Flask 2.0.2
-- Jinja2 3.0.2
-- python-dotenv 0.19.0
-- mysql-connector-python 8.0.26
+- Python 3.10
+- Flask 2.2.3
+- Jinja2 3.1.2
+- python-dotenv 1.0.0
+- mysql-connector-python 8.0.32
 - mysql 8.0  
 
-See the Pipfile for the requirements and dependencies.
+See the Pipfile for the requirements and dependencies. See the Pipfile.lock for the specific versions.
 
-To run the file locally:
-1. Clone the files and folders in this repo.
+### To run the web app locally
 
-2. In the Terminal, start the python web server with:  ` python3 
-   create_jazz_lyric.py `
+1. Clone the repo: `git clone https://github.com/kimlew/pythonapp-createjazzlyric`
 
-3. In the browser, go to: http://localhost:5000/
+2. Change directory: `cd pythonapp-createjazzlyric`
+   
+3. Install pipenv: `pip3 install pipenv`
+
+4. Install required packages and dependencies into the virtual environment with: `pipenv install`
+   
+5. Start the virtual environment: `pipenv shell`
+
+6. In the virtual environment, start the web app with: `python3 create_jazz_lyric.py`
+
+7. In the browser, go to: `http://localhost:5000/`
```

Comment:
This looks good but you're missing instructions for setting up the database. I got the Python application to start but it threw an exception when I attempted to send it a request in the browser because it couldn't connect to the database.

Reply:
@kcharter 
I've added more steps to address setting up and using MySQL, as well as for setting up a Python environment and running the app.

I can see the running app in Firefox but **cannot** see it running in either Opera or Chrome. Both browsers have been updated.
I can't think of any reason for this, nor can I find any info online of anyone else having this weird problem. 😕

Reply:
Later, I did learn in a LinkedIn tutorial that it works in **Firefox** but Chrome (and probably Opera) have some security? barriers that prevent the app from displaying, but the instructor didn't go into further detail.

Reply:
Maybe if you turn on the built-in developer tools in one of those browsers you'll see an error of some kind?

Reply:
I turned on Developer Tools in both Chrome & Opera, but they still only show a blank page without any errors.  :-(